lib/Terminal/LineEditor/RawTerminalInput.rakumod

```diff
@@ -341,11 +341,11 @@ role Terminal::LineEditor::RawTerminalIO {
         # mark the saved termios as unused, and optionally output a \n to push
         # the cursor to the start of the next line
 
-        if $!saved-termios {
+        if $!saved-term-config {
             $!done ⚛= 1;
             my $fd = $.input.native-descriptor;
-            Terminal::MakeRaw::setattr($fd, $!saved-termios, :DRAIN);
-            $!saved-termios = Nil;
+            Terminal::API::restore-config($!saved-term-config, $fd, :when(Terminal::API::DRAIN));
```

Comment:
Any particular reason for switching the `$fd` and `$saved` arguments between the two APIs?

Reply:
The `$fd` arg is optional in T-API (defaulting to `$*IN.native-desctiptor`), thus I need to have it in the second place.

Reply:
AH!  Gotcha, makes sense.